script.js

```diff
@@ -0,0 +1,82 @@
+const form = document.getElementById('form');
+// const list = document.querySelector('.table-part');
```

Comment:
- [ ] Kindly remove commented code.

Reply:
all code comments has been remove from my code base thank you